docs/administering/ssl.md

```diff
@@ -1,29 +1,38 @@
 ## Sandstorm and HTTPS
 
-If you are using a hostname like `example.sandcats.io`, then you likely already have working HTTPS
-(SSL) for your hostname. This page provides details on a variety of options for setting up HTTPS,
-including the `sandcats.io` free certificates.
+Sandstorm can now terminate TLS connections for most conventional configurations you may wish to employ.
+This includes automatic certificate renewal of certificates, if you utilize a supported DNS provider, as well
+as the ability to manually upload your own certificates, either self-signed or from a well-trusted certificate
+vendor.
 
 ### How to get HTTPS on your Sandstorm install
 
-You have a few options.
+If you are using a hostname like `example.sandcats.io`, then you likely already have working HTTPS for your
+hostname. [This page](sandcats-https.md) provides help and advice for enabling and troubleshooting HTTPS for
+Sandcats-based installs.
 
-- Use [sandcats.io free HTTPS](sandcats-https.md), a free service of the Sandstorm.io company. Read
-  details, including how to enable/disable, on that page. Sandstorm automatically renews these
-  certificates, and they are valid in virtually all browsers.
+If you utilize a supported DNS provider (Sandcats.io, Cloudflare, Digital Ocean, DNSimple, Duck DNS, GoDaddy,
+Gandi, Namecheap, Name.com, AWS Route 53, or Vultr), you can create an ACME account from "SSL/TLS Certificates"
```

Comment:
I think I would break this list out into a proper bullet list below; this sentence scans poorly with the huge parenthetical in the middle of it. Maybe end this paragraph with something like "The currently supported DNS providers are:" and then follow with a bullet list.

(This will also make it a bit nicer to edit if/when we add more providers).

Reply:
It feels like... really long to do that this way... but I did it. What do you think?